Maskbook/Scene/App/FIleService/View/FileServiceOptionView.swift

```diff
@@ -0,0 +1,179 @@
+
+                    Section {
+                        ForEach(fileServiceOption.service.options) { item in
+                            Row<Option>(
+                                title: item.title,
+                                item: item,
+                                style: .option,
+                                selection: $fileServiceOption.option
+                            )
+                        }
+                    } header: {
+                        Text(L10n.Plugins.FileService.otherOption)
+                            .font(.bh3)
+                            .foregroundColor(Asset.Colors.Text.dark)
+                            .horizontallyFilled()
+                            .background(Asset.Colors.Background.normal.asColor())
+                    }
+                }
+            }
+
+            Button(
+                action: { optionHandler(fileServiceOption) },
+                label: {
+                    Asset.Colors.Public.blue.asColor()
+                        .overlay(
+                            Text(L10n.Common.Controls.confirm)
+                                .font(.bh5)
+                                .foregroundColor(Asset.Colors.Text.lighter.asColor())
+                        )
+                }
+            )
+            .cornerRadius(8)
+            .frame(height: 54)
+        }
+        .padding(.horizontal, 20)
+        .padding(.top, 20)
+        .background(Asset.Colors.Background.normal.asColor())
+    }
+}
+
+fileprivate struct Row<Selection: Equatable>: View {
+    enum Style {
+        case service
+        case option
+    }
+
+    var selected: Bool {
+        item == selection
+    }
+
+    let title: String
+    let item: Selection
+    let style: Style
+
+    @Binding var selection: Selection
+
+    var body: some View {
+        RoundedRectangle(cornerRadius: 12)
+            .onTapGesture {
+                guard !selected else {
+                    return
+                }
+
+                selection = item
+            }
+            .foregroundColor(Asset.Colors.Background.dark.asColor())
+            .frame(height: 56)
+            .overlay(
+                HStack(spacing: 4) {
+                    Text(title)
+                        .font(.bh5)
+                        .foregroundColor(Asset.Colors.Text.dark)
+                        .horizontallyFilled()
+
+                    switch style {
+                    case .service:
+                        selected
+                        ? Asset.Icon.Cell.cellCheck.asImage()
+                        : Asset.Icon.Cell.cellUncheck.asImage()
+
+                    case .option:
+                        selected
+                        ? Asset.Images.Scene.WalletList.check.asImage()
+                        : Asset.Images.Scene.WalletList.unchecked.asImage()
+                    }
+                }
+                .padding(.horizontal, 12)
+            )
+    }
+}
+
+struct FileServiceOption {
```

Comment:
`FileServiceOption` will be used in other places. I think put it into `Model` is better